Can't build on nodejs 16.0.0

**Environment Information**
 - OS ubuntu 20.04: 
 - Node Version 16.0.0:
 - NPM Version 7.0.0:
 - C++ Toolchain g++:
 - node-rdkafka version 2.1.0:

**Steps to Reproduce**

Just install package

**node-rdkafka Configuration Settings**
No context. Just can't build it coz in nodejs 16 new ver of v8. 

**Additional context**
`
npm ERR! /home/resation/.cache/node-gyp/16.0.0/include/node/v8-internal.h: In function ‘void v8::internal::PerformCastCheck(T*)’:
npm ERR! /home/resation/.cache/node-gyp/16.0.0/include/node/v8-internal.h:452:38: error: ‘remove_cv_t’ is not a member of ‘std’; did you mean ‘remove_cv’?
`
[2021-04-22T16_45_22_674Z-debug.log](https://github.com/Blizzard/node-rdkafka/files/6359955/2021-04-22T16_45_22_674Z-debug.log)

I have the same problem...

**Environment Information**

- OS ubuntu 20.04:
- Node Version 16.0.0:
- NPM Version 7.11.0:
- C++ Toolchain g++:
- node-rdkafka version 2.10.0
- node-gyp 8.0.0

**Steps to Reproduce**

Just install package

node-rdkafka Configuration Settings
No context. Just can't build it coz in nodejs 16 new ver of v8.

Additional context
npm ERR! /home/resation/.cache/node-gyp/16.0.0/include/node/v8-internal.h:452:38: error: ‘remove_cv_t’ is not a member of ‘std’; did you mean ‘remove_cv’?

--- a/.travis.yml
+++ b/.travis.yml
@@ -12,6 +12,7 @@ node_js:
   - "13"
   - "14"
   - "15"
+  - "16"
 sudo: required
 services: docker
 before_install:

--- a/binding.gyp
+++ b/binding.gyp
@@ -43,7 +43,7 @@
               }
             ],
             'cflags_cc' : [
-              '-std=c++11'
+              '-std=c++14'
             ],
             'msvs_settings': {
               'VCLinkerTool': {
@@ -122,7 +122,7 @@
                 'OS=="linux"',
                 {
                   'cflags_cc' : [
-                    '-std=c++11'
+                    '-std=c++14'
                   ],
                   'cflags_cc!': [
                     '-fno-rtti'
@@ -140,7 +140,7 @@
                     ],
                     'OTHER_CPLUSPLUSFLAGS': [
                       '-I/usr/local/opt/openssl/include',
-                      '-std=c++11'
+                      '-std=c++14'
                     ],
                   },
                 }

